conf/DEV.public.conf

```diff
@@ -17,7 +17,14 @@ stripe {
         }
     }
 }
-
+paypal {
+    default=TEST
```

Comment:
I'm not sure that paypal should be split like this instead of using the existing DEV->UAT->PROD split config. (With DEV & UAT -> TEST)